src/amo/components/AddonMoreInfo.scss

```diff
@@ -1,5 +1,5 @@
 @import "~core/css/inc/mixins";
-@import "~core/css/inc/vars";
+@import "~amo/css/inc/vars";
```

Comment:
I guess that makes sense, but I'd think a lot of things would have shared styles between AMO and Disco and DEFINITELY in the future should between the admin and AMO, right? That way they won't slowly drift apart (as they definitely have on addons-server).

Though if we don't have good UI tests for them it'll be a PITA, so I guess I see reasons for both.